app/validators/embargo_date_validator.rb

```diff
@@ -6,6 +6,27 @@ class EmbargoDateValidator < ActiveModel::EachValidator
   def validate_each(record, attribute, value)
     return if value.nil?
 
+    collection = Collection.where(id: record.model[:work][:collection_id]).first
+    return if collection.blank? || collection.release_duration.blank?
+
+    valid_embargo_value(record, attribute, value) if valid_embargo_collection(collection, record, attribute, value)
+  end
+
+  private
+
+  def valid_embargo_collection(collection, record, attribute, value)
+    year_match = collection.release_duration.match(/(\d+)/)
```

Comment:
Since the `collection.release_duration` is no longer a date but a string, this regex seemed to be the best option but open to suggestions on a better method of extract the number of years.